third_party/proton/dialect/lib/ProtonGPUToLLVM/ProtonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -122,13 +122,11 @@ int TargetInfo::getAddressSpace(Attribute addressSpace) const {
   int spaceId = 0;
   if (mlir::isa<triton::gpu::SharedMemorySpaceAttr>(addressSpace)) {
     spaceId = 3;
-  } else if (mlir::isa<proton::gpu::StackMemorySpaceAttr>(addressSpace)) {
-    spaceId = 1;
   } else if (mlir::isa<proton::gpu::GlobalMemorySpaceAttr>(addressSpace)) {
```

Comment:
If going to take this out then also should take out the global memory code too since it's not implemented.